Tribler/Main/Utility/GuiDBHandler.py

```diff
@@ -73,6 +75,10 @@ def Add(self, sender, workerFn, args=(), kwargs={}, name=None, delay=0.0, uId=No
         """The sender will send the return value of
         workerFn(*args, **kwargs) to the main thread.
         """
+        if self.quitting:
+            self._logger.debug("GUIDBHandler: rejecting task(%s), quitting flag is set.", name)
```

Comment:
Also, not necesary to prepend the class names on the log entries, the logging system takes care of this kind of things.